python/cugraph/cugraph/sampling/node2vec.py

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pylibcugraph
+import cudf
+from cugraph.utilities import ensure_cugraph_obj_for_nx
+
+
+def node2vec(G, start_vertices, max_depth, use_padding, p=1.0, q=1.0):
```

Comment:
No default value for both max_depth and use_padding which are both optional. If the user doesn't pass either or both, wouldn't you get a `required positional argument missing error`?